rigs/heft-web-rig/shared/webpack-base.config.js

```diff
@@ -0,0 +1,270 @@
+      ? {
+          chunkFilename: '[name].[contenthash].js',
+          filename: '[name].[contenthash].js',
+          sourceMapFilename: '[name].[contenthash].js.map'
+        }
+      : {},
+    module: {
+      rules: [
+        {
+          // The source-map-loader extracts existing source maps from all JavaScript entries. This includes both
+          // inline source maps as well as those linked via URL. All source map data is passed to Webpack for
+          // processing as per a chosen source map style specified by the devtool option in webpack.config.js.
+          // https://www.npmjs.com/package/source-map-loader
+          test: /\.js$/,
+          include: [path.resolve(projectRoot, 'lib')],
+          enforce: 'pre',
+          use: [
+            {
+              loader: require.resolve('source-map-loader')
+            }
+          ]
+        },
+
+        {
+          // Recommendation for CSS styles:
+          //
+          // We recommend the newer .scss file format because its syntax is a proper superset of familiar CSS.
+          // The older .sass syntax is accepted for backwards compatibility.
+          //
+          // Handling of formats:
+          // - *.scss:          sass, autoprefixer, modules  <--- recommended
+          // - *.global.scss:   sass, autoprefixer, NO modules
+          // - *.css:           NO sass, NO autoprefixer, NO modules
+          //
+          // If someone wants CSS with modules, they can simply use the .sass file extension, since SASS
+          // is essentially a superset of CSS. (There's a small performance penalty for applying SASS to
+          // a plain CSS file, but the extra syntax validation probably justifies that cost.)
+          //
+          // The SASS docs are here: https://sass-lang.com/documentation/syntax
+          test: /\.(scss|sass|css)$/,
+          exclude: /node_modules/,
+          use: [
+            // "For production builds it's recommended to extract the CSS from your bundle being able to
+            // use parallel loading of CSS/JS resources later on. This can be achieved by using the
+            // mini-css-extract-plugin, because it creates separate css files."
+            //
+            // "For development mode (including webpack-dev-server) you can use style-loader, because it injects
+            // CSS into the DOM using multiple <style></style> and works faster."
+            //
+            // "WARNING: Do not use style-loader and mini-css-extract-plugin together."
+            production
+              ? {
+                  loader: MiniCssExtractPlugin.loader
+                }
+              : {
+                  // Generates JavaScript code that injects CSS styles into the DOM at runtime.
+                  // The default configuration creates <style> elements from JS strings
+                  // https://www.npmjs.com/package/style-loader
+                  loader: require.resolve('style-loader')
+                },
```

Comment:
Above is how I interpret the recommended guidance from the Webpack docs.  Are you using 'mini-css-extract-plugin'?